nx/include/switch/services/nfc.h

```diff
@@ -6,8 +6,7 @@
  */
 
 #pragma once
-#include "../types.h"
-#include "../sf/service.h"
+#include "mii.h"
```

Comment:
Keep the original includes, and use: `#include "../services/mii.h"` (same with miiimg.h)

Reply:
Done